src/pages/Details.js

```diff
@@ -0,0 +1,69 @@
+import moment from 'moment';
+import {
+  CardList,
+  CardText,
+  CenteredTextContainer,
+  ColWrapper,
+  RowWrapper,
+  WeatherCard
+} from '../components/styledComponents';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/punny-weather/pullRequest?prid=7693782)
```suggestion
} from "../components/styledComponents";
```